specifyweb/frontend/js_src/lib/components/Toolbar/RecordSets.tsx

```diff
@@ -79,6 +80,8 @@ export function RecordSetsDialog({
 
   const [data] = useAsyncState(
     React.useCallback(
+      /** domainFilter does filter for tables that are
```

Comment:
did you mean to say "does **not** filter"?

Reply:
This comment is technically correct, assuming it means the backend does correctly scope tables which only have the `collectionmemberid` field:

https://github.com/specify/specify7/blob/c98c9c3341fc72a9610ee65e7689180a8e6a31e0/specifyweb/specify/filter_by_col.py#L36-L39

Though we have to manually specify the `collectionmemberid` here because the frontend currently does not handle the case where a table is only scoped by the `collectionmemberid` field (see https://github.com/specify/specify7/pull/4542#pullrequestreview-1894657939 and a list of tables which also exhibit this bug in https://github.com/specify/specify7/pull/4542#issuecomment-1959756209)

The comment can likely be expanded to include a little more context, if needed